README.md

```diff
@@ -1,87 +1,49 @@
-![MobileCoin logo](https://raw.githubusercontent.com/mobilecoinofficial/mobilecoin/master/img/mobilecoin_logo.png)
+---
```

Comment:
I think we want this to be one layer down - so that we can keep the current README - or rename the file to INTEGRATION_GUIDE.md (or something similar)